multi_line_stringm_test.go

```diff
@@ -0,0 +1,83 @@
+package geom_test
+
+import (
+	"reflect"
+	"strconv"
+	"testing"
+
+	"github.com/go-spatial/geom"
+)
+
+func TestMultiLineStringMSetter(t *testing.T) {
+	type tcase struct {
+		pointms  [][][3]float64
+		setter   geom.MultiLineStringMSetter
+		expected geom.MultiLineStringMSetter
+		err      error
+	}
+	fn := func(t *testing.T, tc tcase) {
+		err := tc.setter.SetLineStringMs(tc.pointms)
+		if tc.err == nil && err != nil {
+			t.Errorf("error, expected nil got %v", err)
+			return
+		}
+		if tc.err != nil {
+			if tc.err.Error() != err.Error() {
+				t.Errorf("error, expected %v got %v", tc.err, err)
+			}
+			return
+		}
+
```

Comment:
I've started to change the way we write tests a bit:
instead of 

```go

fn := func(t *testing.T, tc tcase) {

...

```

I'm not starting to change all the tests to look like:

```go

fn := func(tc tcase)func(*testing.t){
    return func(t *testing.T){
     ...
     }
 }
 ```
 
 then  for loop below becomes:
 
 ```go
 
for i := range tests {
   t.Run(strconv.FormatInt(int64(i), 10), fn(tests[i])) 
} 

```

this change would have to apply to all the tests you have in the old style.

Reply:
Here is an example:
https://github.com/go-spatial/geom/blob/master/geom_math_test.go

Reply:
@gdey I changed the tests in order to use the decorators, following the new style. Ready for a new review.